README.md

```diff
@@ -13,7 +13,7 @@ Contributing and Communications
 * The [Issues and Milestones](https://github.com/meteor-gazelle/meteor-gazelle/issues) contains items that need to be worked on. Check there for a general overview of what needs to be worked on.
 * IRC
   * &#35;gazelle on irc.what-network.net is the official meteor-gazelle development channel.
-  * If you are looking for something to work on, ping a Core team member (operators in #gazelle).
+  * If you are looking for something to work on, ping the core team using !core <message>.
```

Comment:
We're forgetting about creating issues, lets not limit technical discussion to tribal knowledge within the confines of #gazelle, just cause we idle in there doesn't mean an outside pull request contributor would.

Join irc and ping or create a gitub issue labeled appropriately...